mifosng-android/src/main/java/com/mifos/mifosxdroid/dialogfragments/documentdialog/DocumentDialogFragment.java

```diff
@@ -299,28 +320,51 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
 
 
     @Override
-    public void showDocumentedCreatedSuccessfully(GenericResponse genericResponse) {
-        Toast.makeText(getActivity(), String.format(getString(R.string
-                        .uploaded_successfully), fileChoosen.getName()),
-                Toast.LENGTH_SHORT).show();
+    public void showDocumentedCreatedSuccessfully(DocumentRelatedResponse
+                                                              documentCreationResponse) {
+        if (onDocumentCreatedListener != null) {
+            createdDocument.setId(documentCreationResponse.getResourceId());
```

Comment:
The View should know nothing about the data ☹️

Reply:
Can we have a discussion regarding this, please?